pkg/alt/knative/pkg/webhook/resourcesemantics/conversion/README.md

```diff
@@ -0,0 +1,8 @@
+This code is all copied from https://github.com/knative/pkg/tree/main/webhook/resourcesemantics/conversion
+
+The only changes were made to reconciler.go to support `crd.spec.conversion.webhook.clientConfig.url` in addition
+to `crd.spec.conversion.webhook.clientConfig.service`.
+
+Look for the sections that start with `// AKS customized` to understand the changes.
+
+Also note that the zz_generated.deepcopy.go isn't currently actually autogenerated it's just copied from upstream.
```

Comment:
happy to fix this if you point me to where to do so.

Reply:
I would just leave it as is; just copying it should be fine. And it seems to be safely out of the way of code generation we do run ...